File: Python/Inserción_al_inicio.py
arreglo = [10, 20, 30, 0, 0, 0, 0, 0, 0, 0]
n = 3

def ins_inicio(valor):
    global n 
    for i in range(n, 0, -1):             
        arreglo[i] = arreglo[i - 1]        
    arreglo[0] = valor
    n += 1

ins_inicio(1)
print(arreglo[:n]) #para que solo muestre los que tienen valores en el arreglo

File: JavaScript/Inserción_al_inicio.js
let arreglo = [10, 20, 30, 0, 0, 0, 0, 0, 0, 0];
let n = 3;

function insInicio(valor) {
    for (let i = n; i > 0; i--) {
        arreglo[i] = arreglo[i - 1];
    }
    arreglo[0] = valor;
    n++;
}

insInicio(5);
console.log(arreglo.slice(0, n));